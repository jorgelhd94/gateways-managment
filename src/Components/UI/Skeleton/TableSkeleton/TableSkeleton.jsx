import React from 'react';

const TableSkeleton = () => {
  return (
    <div className="bg-white rounded w-full mx-auto rounded-2xl mt-8">
      <div className="bg-gray-200 h-12 p-3 overflow-hidden animate-pulse"></div>
      <div className="">
        <div className="grid grid-cols-3 gap-4 mt-2">
          <div className="h-8 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-8 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-8 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-8 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-8 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-8 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-8 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-8 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-8 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-8 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-8 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-8 bg-gray-200 rounded animate-pulse"></div>
          </div>
      </div>
    </div>
  );
};

export default TableSkeleton;
