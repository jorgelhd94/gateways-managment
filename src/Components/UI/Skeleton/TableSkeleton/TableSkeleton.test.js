import React from 'react';
import renderer from 'react-test-renderer';
import TableSkeleton from './TableSkeleton';

test('TableSkeleton render correctly', () => {
  const component = renderer.create(<TableSkeleton />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
