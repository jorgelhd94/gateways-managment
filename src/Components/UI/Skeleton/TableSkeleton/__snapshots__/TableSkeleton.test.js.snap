// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableSkeleton render correctly 1`] = `
<div
  className="bg-white w-full mx-auto rounded-2xl mt-8"
>
  <div
    className="bg-gray-200 h-12 p-3 overflow-hidden animate-pulse"
  />
  <div
    className=""
  >
    <div
      className="grid grid-cols-3 gap-4 mt-2"
    >
      <div
        className="h-8 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-8 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-8 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-8 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-8 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-8 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-8 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-8 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-8 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-8 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-8 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-8 bg-gray-200 rounded animate-pulse"
      />
    </div>
  </div>
</div>
`;
